.devcontainer/Dockerfile

```
@@ -0,0 +1,74 @@
+# Author: Jonas Vautherin
+# Original Found at: https://github.com/JonasVautherin/px4-gazebo-headless
+# Edited by: David Shearon
+
+FROM ubuntu:18.04
```

We should really change this to ubuntu:20.04 I guess I missed this before

```suggestion
FROM ubuntu:20.04
```